guides/Getting Started/GettingStarted.mdx

```diff
@@ -0,0 +1,25 @@
+import { saphFrameworkUrl, djsUrl, createDjsTypedefUrl } from '../constants.mdx';
+
+export const djsClientOptions = createDjsTypedefUrl('ClientOptions');
+
+# Welcome to the @sapphire/framework#{@branch} documentation
+
+## Installing <a href={saphFrameworkUrl}>@sapphire/framework</a>
+
+You can install this version of @sapphire/framework with one of the following commands:
+
+```sh
+npm install --save @sapphire/framework
```

Comment:
I would personally separate these and annotate that you use command x when you're using package manager y.

This is just to prevent "copy pasta without reading"-people to copy the whole block and paste it expecting it to work.

Reply:
Or maybe even leave out the other options, most people that are using a different option know that `npm install` is the same as `yarn add`